akd/src/storage/cache/high_parallelism.rs

```diff
@@ -34,6 +34,8 @@ pub struct TimedCache {
     can_clean: Arc<AtomicBool>,
     item_lifetime: Duration,
     memory_limit_bytes: Option<usize>,
+
+    #[cfg(feature = "runtime_metrics")]
```

Comment:
Doing this, you need to do it everywhere it's referenced which breaks a lot of stuff. Essentially what I had done, was to create the field but only populate/update it when the feature is enabled. That way it's just a non-accessed 0-value.

Reply:
Ah yeah, but there was a 'is never read' warning and I was hoping to address it 

<img width="580" alt="image" src="https://user-images.githubusercontent.com/20502803/209890008-e0c0523b-6400-401a-b299-2b8ea260581f.png">